### PR TITLE
♻️ Move http blocker parameters from NodeEnv to "startRuntime" arguments

### LINKED ISSUE #461: Clarify naming for "/" route
## Description

Das Wording von `DO_NOT_BLOCK_GLOBAL_ROUTE` finde ich extrem unglücklich.

* Hier wird von der PE aus ihrer Sicht nichts in ihrer Fachlichkeit "blockiert". Wir beschreiben mit diesem Namen vielmehr einen möglichen Schmerz des Endnutzers, der ihn dazu bringen könnte, nach diesem Parameter zu suchen.
* Auch den Ausdruck "globale Route" finde ich nachfragebedürftig (habe ich so als Synonym für "/" auch noch nicht gehört).
* Die Negativ-Formulierung des Parameters finde ich nicht gut. Meiner Meinung nach ist es passender, ein Verhalten abzuschalten, indem man etwas auf `false` setzt (anstatt eine negativ formulierte Variable auf `true` zu setzen).
* Die Tatsache, dass man den Parameter auf irgendeinen Wert setzen kann, finde ich nicht explizit und im krassen Gegensatz zu der sonst üblichen `x === null || x === undefined`-Pedantik.

Ich frage mich, wieso es einen "LASS_DAS"-Parameter gibt und eine hard-codierte Route `/process_engine`. Das ist aus meiner Sicht "technical debt mit Ankündigung".

Wäre es nicht viel passender, wenn ich einfach die Route, unter der alles ausgeliefert werden soll, konfigurieren könnte? Bspw. `PARAMETER_FUER_ROUTE='/my_pe'` und dann trage ich im Studio einfach `localhost:5000/my_pe` ein?

Wir sollten das vor dem Stable-Release klären. 👍

### PR DESCRIPTION
**Pull request by ElRaptorus**
## Changes

1. Remove `NO_HTTP` and `DO_NOT_BLOCK_GLOBAL_ROUTES` environment parameters
2. Accept `enableHttp` and `useHttpRootRoutes` as optional parameters for `startRuntime`
    - In essence, these will do what the removed environment variables did, but with a default positive value
3. Update the Readme
4. Update the tests for the root http routes
5. Add missing `types` section to `package.json`

## Issues

Closes #461 

PR: #466

## How to test the changes

- The Node env parameters `NO_HTTP` and `DO_NOT_BLOCK_GLOBAL_ROUTES` should not work anymore
- The parameters for `startRuntime()` should now accept two additional optional parameters:
    - `enableHttp`: Enable/Disable all http endpoints. Defaults to `true`
    -  `useHttpRootRoutes`: Enable/Disable usage of `/` and `/security/authority` routes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,14 +165,21 @@ await ProcessEngine.startRuntime(args);
...
-  Use this, if you use a customized startup that is not compatible with the runtime's own startup routine
+- `logFilePath`: A path to where the runtime should store its logfiles. The path must be absolute
+- `container`: An `addict-ioc` InvocationContainer, where the runtime should register its dependencies at
+- `minimalSetup`: If set to true, the runtime will only perform ioc registrations, but nothing else
+    - Use this, if you want to launch the ProcessEngineRuntime manually
+    - Defaults to `false`
+- `enableHttp`: If set to true, all HTTP endpoints the ProcessEngineRuntime uses will be loaded
+    - Use `false` to prevent the ProcessEngineRuntime from providing HTTP endpoints
+    - Defaults to `true`
+- `useRootRoutes`: If set to `true`, the routes `/` and `/security/authority` will be set by the ProcessEngineRuntime
```
Comment (rrrene) [inline README.md]: Wird der Name durch `useHttpRootRoutes ` sprechender?

File: src/main.ts [hunk 2]
```diff
@@ -34,9 +34,12 @@ type startupArgs = {
   container?: InvocationContainer;
   minimalSetup?: boolean;
   workDir?: string;
+  enableHttp?: boolean;
+  useHttpRootRoutes?: boolean;
```
Comment (rrrene) [inline src/main.ts]: 😳 Es ist, als könntest du Gedanken lesen ^_^ (s.o.)

Comment (ElRaptorus) [reply src/main.ts]: Wer weis... 👀

**Comment by ElRaptorus** [comment]
@rrrene done

**Pull request merged**